lib/mayaHydra/hydraExtensions/sceneIndex/mayaHydraSceneIndex.h

```diff
@@ -203,10 +203,16 @@ class MAYAHYDRALIB_API MayaHydraSceneIndex : public HdRetainedSceneIndex, public
 
     bool GetPlaybackRunning() const;
 
-    void SetDefaultMaterial(bool useDefMaterial);
-
     Fvp::PrimSelections UfePathToPrimSelections(const Ufe::Path& appPath) const override;
 
+    //Sdfpath of the maya default material
+    SdfPath GetDefaultMaterialPath() const{return _mayaDefaultMaterialPath;}
+
+    bool GetDefaultMaterialAlreadyCreated() const{return _defaultMaterialCreated;}
```

Comment:
Accessor name is a bit wordy, I like following the name of the member variable `DefaultMaterialCreated()`, or if you really want `IsDefaultMaterialAlreadyCreated()`